diff --git a/jre/java/javaemul/internal/vmbootstrap/primitives/generate_primitive.bzl b/jre/java/javaemul/internal/vmbootstrap/primitives/generate_primitive.bzl
index 910ebc0..46bf4bf 100644
--- a/jre/java/javaemul/internal/vmbootstrap/primitives/generate_primitive.bzl
+++ b/jre/java/javaemul/internal/vmbootstrap/primitives/generate_primitive.bzl
@@ -5,8 +5,8 @@ def generate_primitive_type(name, **kwargs):
   """Generates src for J2CL primitive types"""
 
   # exit early to avoid parse errors when running under bazel
-  if not hasattr(native, "js_library"):
-    return
+#  if not hasattr(native, "js_library"):
+#    return
 
   templates = ["primitive.java.js", "primitive.impl.java.js"]
   native.filegroup(
